src/_modules/footer/footer.jade

```diff
@@ -1,13 +1,16 @@
 footer.footer
-
-        .footer__container
-            .footer__container__contact
-                i.fa.fa-phone
-                a(href="+54 11 4708-4108") 4708-4108
-                span Vos tambien podes sumarte!
-                .footer__container__contact__buttons
-                    a VOLUNTARIADO
-                    a DONACIONES
-                            
-            .footer__container__copyright
-                span La merced vida &copy; 2017, Todos los derechos reservados
+                    
+        .footer__contact__container
+          
+          .footer__contact__container__data
+            i.fa.fa-phone
+            a.footer__contact__container__data__phone(href="+54 11 4708-4108") 4708-4108
+            span.footer__contact__container__data__text Vos tambien podes sumarte!
+            
+          .footer__contact__buttons
+            .footer__contact__buttons__container
```

Comment:
- .footer__contact__buttons__container is unnecessary, assign class to anchors

Reply:
Done